reproduce/face_detector.py

```diff
@@ -73,9 +51,16 @@ def extract_bboxes(face_group_entry):
             if isinstance(face[0], tuple):
                 face = list(face[0])
             bbox = face[0]
+            bbox[0] = max(bbox[0], 0)  # left side of box
+            bbox[1] = max(bbox[1], 0)  # top side of box
```

Comment:
Would there be a problem in a case where the bounding box is the entire frame?

Reply:
Nope, that should work fine!

Reply:
Perfect, then I think we're good here!